services/scraper/search_keyword.go

```diff
@@ -0,0 +1,86 @@
+package scraper
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/beego/beego/v2/core/logs"
+	"github.com/gocolly/colly/v2"
+)
+
+type SearchKeywordService struct {
+	Keyword string
+
+	isSynchronous bool
+	searchResult  *searchKeywordResult
+}
+
+type searchKeywordResult struct {
```

Comment:
![Screen Shot 2021-02-25 at 10 48 54 AM](https://user-images.githubusercontent.com/6076762/109100235-1a2dc780-7757-11eb-85c5-09ee4299cc6e.png)

Could we add an attribute for counting total link items and an attribute for raw html?

Reply:
@junan for HTML code and other the rest I'll add it in this ticket (https://github.com/Lahphim/go-crawler-challenge/issues/15)